va/va_vpp.h

```diff
@@ -510,11 +510,15 @@ typedef struct _VABlendState {
  * \brief Specifies whether to apply power or performance
  * optimizations to a pipeline.
  *
- * When processing several surfaces, it may be necessary to prioritize
- * more certain pipelines than others. This flag is only a hint to the
- * video processor so that it can omit certain filters to save power
- * for example. Typically, this flag could be used with video surfaces
- * decoded from a secondary bitstream.
+ * This flag is a hint to the video processor to achieve faster processing speed
+ * (better performance)in the pipeine. Hardware vendor might have several approaches
+ * for one functionality, this flag is to indicate the one with best performance.
+ * Performance depends on the hardware capablity, plesae refer to the specific hardware
+ * specification from hardware vendors. Here, lists a few examples as the reference.
+ * 1) When processing several surfaces, it may be necessary to prioritize more certain
+ * pipelines with this flag than others. 2) This flag is only a hint to the video processor
+ * so that it can omit certain filters for better performance for example. 3) Typically, this
+ * flag could be used with video surfaces decoded from a secondary bitstream.
```

Comment:
I think we should give more description about the relationship between this flag and the VA_FILTER_SCALING_FAST and VA_FILTER_SCALING_HQ flag here.